src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1428,10 +1431,10 @@ codet java_bytecode_convert_methodt::convert_instructions(
 
       assert(arg0.id()==ID_virtual_function);
 
-      // does the function symbol exist?
+      // does the function symbol exist? is it virtual and inherited?
```

Comment:
I find the comment unclear as there are two questions. Could we replace this with:
```
// create a stub for functions whose symbol does not exist
// and that are neither virtual nor inherited
```
But we might want to be more explicit about why we do that (and that I do not understand fully).
I find it a bit dangerous to rely on the existence of a stub to decide whether a method should be treated as non-det. Isn't there a more direct way?